### PR TITLE
Add support for strings

### PR DESCRIPTION
**Pull request by brian-brazil**
This exposes strings as a label value on a 1-valued gauge.

@SuperQ @RichiH

### CODE REVIEW

File: generator/tree.go [hunk 1]
```diff
@@ -74,6 +74,11 @@ func prepareTree(nodes *Node) map[string]*Node {
 
 	// Set type on MAC addresses.
 	walkNode(nodes, func(n *Node) {
+		// For some odd reason ifPhysAddress's MIB isn't bein parsed correctly
```
Comment (RichiH) [inline generator/tree.go]: Being.

**Comment by RichiH** [comment]
Thanks; I just skimmed over it and will try to find time to test over the weekend.

File: generator/tree.go (hunk 1 above)
Comment (brian-brazil) [inline generator/tree.go]: That's the Dublin accent finally getting to me :)

**Comment by RichiH** [comment]
Belated "my tests looked good".

**Pull request merged**